src/scenes/enemy-level.js

```diff
@@ -172,14 +177,28 @@ export class EnemyLevelScene extends LevelScene {
     getDamageLevel() {
         switch(this.characterData.characterClass) {
             case CONSTANTS.CLASS.MAGE:
-                return this.calcLevel(this.characterData.skills.magic);
+                return calcLevel(this.characterData.skills.magic);
                 break;
             case CONSTANTS.CLASS.RANGER:
-                return this.calcLevel(this.characterData.skills.ranged);
+                return calcLevel(this.characterData.skills.ranged);
                 break;
             case CONSTANTS.CLASS.WARRIOR:
-                return this.calcLevel(this.characterData.skills.attack);
+                return calcLevel(this.characterData.skills.attack);
                 break;
         }
     }
+
+    // Show/hide text
+    showQuestText(show) {
```

Comment:
NIT: can we change the parameter name in both of these methods to isShown or something more indicative of a boolean?

Reply:
Got it